sequencing-server/src/lib/batchLoaders/expansionSetBatchLoader.ts

```diff
@@ -62,15 +58,53 @@ export const expansionSetBatchLoader: BatchLoader<
       if (expansionSet === undefined) {
         return new ErrorWithStatusCode(`No expansion_set with id: ${expansionSetId}`, 404);
       }
-      const commandTypes = await fs.promises.readFile(
-        expansionSet.command_dictionary.command_types_typescript_path,
-        'utf8',
+      const commandTypes = await fs.promises.readFile(expansionSet.parcel.command_dictionary.path, 'utf8');
+      const channelTypes = JSON.parse(
+        expansionSet.parcel.channel_dictionary
+          ? await fs.promises.readFile(expansionSet.parcel.channel_dictionary.path, 'utf8')
+          : '{}',
       );
+      const paramerterTypes = await Promise.allSettled(
+        expansionSet.parcel.parameter_dictionary.map(async param => {
+          return {
+            parameter_dictionary: {
+              parsedJson: JSON.parse(await fs.promises.readFile(param.parameter_dictionary.path, 'utf8')),
+              id: param.parameter_dictionary.id,
+            },
+          };
+        }),
+      );
+
+      const paramerterResults: {
```

Comment:
Minor: typo
```suggestion
      const parameterResults: {
```

Reply:
sigh...those typos always get me. I fixed it.